src/Control/Email/Email.php

```diff
@@ -292,13 +292,26 @@ public function getFrom()
         return $this->getSwiftMessage()->getFrom();
     }
 
+    /**
+     * @param string|array $address
+     * @return string|array
+     */
+    private function trimAddress($address)
```

Comment:
Should we call it `sanitiseAddress` and trim can be one of the actions taken on the items, but maybe not the only one going forward? Is there any other stuff we could/should do here?

Reply:
Updated